pulp_2to3_migration/app/plugin/content.py

```diff
@@ -283,6 +283,8 @@ def get_remote_by_importer_id(importer_id):
                     # find already created pulp3 content
                     pulp3content = pulp2content.pulp3_content
                     extra_info = None
+                    if is_multi_artifact:
+                        extra_info = pulp_2to3_detail_content.get_treeinfo_serialized()
```

Comment:
what about moving this line after https://github.com/pulp/pulp-2to3-migration/blob/5eaba43c5ffb414acf678c037a410aca0af0db9d/pulp_2to3_migration/app/plugin/content.py#L296

Reply:
I think the problem is line 293. Might be some way to rearrange it though.

Reply:
right